### PR TITLE
Confidently testing redux applications with jest typescript notes

### PR DESCRIPTION
**Pull request by meganreckner**

### CODE REVIEW

File: courses/confidently-testing-redux-applications-with-jest-typescript-16e17d9b/notes/02_redux-exploring-the-sample-app-for-the-redux-testing-course.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Exploring the Sample App for the Redux Testing Course
+
+<TimeStamp start="0:15" end="0:30">
+
+`git clone https://github.com/xjamundx/redux-shopping-cart.git redux-testing` to create a folder specifically for this course and clone the repo into it.
+
+</TimeStamp>
+
+<TimeStamp start="0:15" end="0:50">
```
Comment (NicollGuarnizo) [inline courses/confidently-testing-redux-applications-with-jest-typescript-16e17d9b/notes/02_redux-exploring-the-sample-app-for-the-redux-testing-course.md:9]: starting time is the same as previous

**Review by NicollGuarnizo — COMMENTED** [review]
I left some minor comments on syntax, mdx can be very picky sometimes. Other than that looks great! ✨ @meganreckner

File: courses/confidently-testing-redux-applications-with-jest-typescript-16e17d9b/notes/02_redux-exploring-the-sample-app-for-the-redux-testing-course.md (hunk 1 above)
Comment (meganreckner) [reply courses/confidently-testing-redux-applications-with-jest-typescript-16e17d9b/notes/02_redux-exploring-the-sample-app-for-the-redux-testing-course.md]: Fixed this as part of 3626de9

File: courses/confidently-testing-redux-applications-with-jest-typescript-16e17d9b/notes/02_redux-exploring-the-sample-app-for-the-redux-testing-course.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+`cd redux-testing/` to open the project folder
```
[the remark this answers is not in the document]
Comment (meganreckner) [reply courses/confidently-testing-redux-applications-with-jest-typescript-16e17d9b/notes/02_redux-exploring-the-sample-app-for-the-redux-testing-course.md]: Fixed in 1a8c30f.

File: courses/confidently-testing-redux-applications-with-jest-typescript-16e17d9b/notes/02_redux-exploring-the-sample-app-for-the-redux-testing-course.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+`git checkout testing` to check out the branch we'll be working off of
+`npm install`
+`npm run dev` to launch the application we'll be testing
+Open your browser and navigate to `localhost:3000`
+
+</TimeStamp>
+
+<TimeStamp start="0:60" end="1:05">
+
+src/App.tsx contains the three main routes of the application.
```
[the remark this answers is not in the document]
Comment (meganreckner) [reply courses/confidently-testing-redux-applications-with-jest-typescript-16e17d9b/notes/02_redux-exploring-the-sample-app-for-the-redux-testing-course.md]: Fixed in 1a8c30f.

**Pull request merged**